dvc/utils/serialize/__init__.py

```diff
@@ -14,6 +14,13 @@
     {".toml": load_toml, ".json": load_json, ".py": load_py}  # noqa: F405
 )
 
+
+def load_path(fs_path, fs):
```

Comment:
We have `modify_data` in `dvc.utils.serialize._common`. Let's rename this to `load_data` as well.

Reply:
Also let's move this to `dvc.utils.serialize._common`.